src/rest_framework_jwt/authentication.py

```diff
@@ -89,11 +89,11 @@ def authenticate(self, request):
 
     @classmethod
     def get_token_from_request(cls, request):
-        authorization_header = force_str(get_authorization_header(request))
-
         try:
+            authorization_header = force_str(get_authorization_header(request))
+
             return cls.get_token_from_authorization_header(authorization_header)
-        except InvalidAuthorizationCredentials:
+        except (InvalidAuthorizationCredentials, UnicodeDecodeError):
```

Comment:
I find all of these auth plugins to be lacking any logging, especially debug info.  All the apps catch exceptions and defer to other apps, and therefore none of them care about failures, so exceptions (like this one) can be happening and there is no way to debug it by turning up the logging.

Should we add logging?  And then log at level 'info' for this `UnicodeDecodeError`?